Document/0x05d-Testing-Data-Storage.md

```diff
@@ -33,13 +33,34 @@ The following code snippets demonstrate bad practices that disclose sensitive in
 The SharedPreferences API is commonly used to permanently save small collections of key-value pairs. Data stored in a SharedPreferences object is written to a plain-text XML file. The SharedPreferences object can be declared world-readable (accessible to all apps) or private.
 Misuse of the SharedPreferences API can often lead to exposure of sensitive data. Consider the following example:
 
+Example for Java:
+
 ```java
 SharedPreferences sharedPref = getSharedPreferences("key", MODE_WORLD_READABLE);
 SharedPreferences.Editor editor = sharedPref.edit();
 editor.putString("username", "administrator");
 editor.putString("password", "supersecret");
 editor.commit();
 ```
+Example for Kotlin:
+
+```kotlin
+var sharedPref = getSharedPreferences("key", Context.MODE_WORLD_READABLE)
+var editor = sharedPref.edit()
+editor.putString("username", "administrator")
+editor.putString("password", "supersecret")
+editor.commit()
+```
+Example for Dart:
+
+```dart
+storeCredentials() async {
```

Comment:
hi @B3nac , isn't Dart only supported by Flutter? or is it supported by the normal android runtime as well?